docs/REST API/rest-api.md

```diff
@@ -7566,12 +7566,13 @@ Deletes an existing script.
 
 | Name            | Type    | In    | Description                                                                                                                   |
 | --------------- | ------- | ----- | ----------------------------------------------------------------------------------------------------------------------------- |
+| team_id         | integer | query | _Available in Fleet Premium_. The ID of the team to filter scripts. If not specified, it will filter only scripts that are available to hosts with no team. |
```

Comment:
```suggestion
| team_id         | integer | query | _Available in Fleet Premium_. The ID of the team to filter scripts by. If not specified, it will filter scripts that are available to hosts with no team. |
```